scripts/lib/CIME/case/case_run.py

```diff
@@ -42,17 +41,6 @@ def _pre_run_check(case, lid, skip_pnl=False, da_cycle=0):
     # load the module environment...
     case.load_env(reset=True)
 
-    # set environment variables
-
-    if batchsubmit is None or len(batchsubmit) == 0:
```

Comment:
Any idea what these environment variables were for?

Reply:
not at the moment